default.nix

```diff
@@ -26,8 +26,9 @@ rec {
   env = pkgs.buildEnv {
     name = "wire-server-deploy";
     paths = with pkgs; [
-      ansible_2_11
       pythonForAnsible
+      ansible_2_15
```

Comment:
Small hint - though it's not very bad: Usually people try to keep the change set (amount of changed lines) small. I.e. I would have just replaced the `1` with a `5`.

But, this is easy to grasp and fine for now.

Reply:
makes sense, thanks for the suggestion :)
Was unintentional change, I appreciate this suggestion for my long term learning :)